public/app/features/provisioning/Wizard/BootstrapStep.tsx

```diff
@@ -268,13 +171,15 @@ export function BootstrapStep({ onOptionSelect, onStepUpdate }: Props) {
                     <Icon name="info-circle" />
                   </Tooltip>
                 </Stack>
-                <Stack direction="row" gap={2} alignItems="center">
-                  <Switch {...register('migrate.history')} defaultChecked={true} />
-                  <Text>Include history</Text>
-                  <Tooltip content="Include complete dashboard version history" placement="top">
-                    <Icon name="info-circle" />
-                  </Tooltip>
-                </Stack>
+                {repoType === 'github' && settingsData?.legacyStorage && (
```

Comment:
History is only supported when moving from legacy storage (for now) -- and not supported in non versioned file systems